extensions/mssql/src/contracts.ts

```diff
@@ -1575,7 +1575,7 @@ export namespace GetServerContextualizationRequest {
 	export const type = new RequestType<ServerContextualizationParams, azdata.contextualization.GetServerContextualizationResult, void, void>('metadata/getServerContext');
 }
 
-// ------------------------------- < Database Server Contextualization API > ------------------------------------
```

Comment:
I missed removing the word database from this comment block when I was generalizing the contextualization provider.